engine/render/src/render/render.h

```diff
@@ -122,11 +122,11 @@ namespace dmRender
 
     // Takes the contents of the render list, sorts by view and inserts all the objects in the
     // render list, unless they already are in place from a previous call.
-    Result DrawRenderList(HRenderContext context, HPredicate predicate, HNamedConstantBuffer constant_buffer);
+    Result DrawRenderList(HRenderContext context, HPredicate predicate, HNamedConstantBuffer constant_buffer, const dmVMath::Matrix4* frustum_matrix);
 
     Result Draw(HRenderContext context, HPredicate predicate, HNamedConstantBuffer constant_buffer);
-    Result DrawDebug3d(HRenderContext context);
-    Result DrawDebug2d(HRenderContext context);
+    Result DrawDebug3d(HRenderContext context, const dmVMath::Matrix4* frustum_matrix);
+    Result DrawDebug2d(HRenderContext context); // Deprecated
```

Comment:
When we finally get to do a big breaking change update, I hope we get to remove stuff like this!

Reply:
Well, my plan is to remove it now. They haven't been able to use it in a long while anyways.